mythril/support/truffle.py

```diff
@@ -31,6 +31,7 @@ def analyze_truffle_project(args):
             try:
                 name = contractdata['contractName']
                 bytecode = contractdata['deployedBytecode']
+                filename = contractdata['sourcePath'].split('/')[-1]
```

Comment:
Looking at some other code I learned about  [pathlib](https://docs.python.org/3.5/library/pathlib.html)  in Python 3.5+. 

```python
import pathlib
>>> pathlib.Path("/a/b/c/foo.bar").name
'foo.bar'
```

Old Python/Unix folks know this function/command as `basename`.

Reply:
Does the pathlib work for windows?( i think the slashes are opposite for windows if thats the case the split i wrote will fail)

Reply:
Excellent point! It looks like you've checked .